azext_iot/operations/central.py

```diff
@@ -30,6 +34,15 @@ def iot_central_device_show(
         raise CLIError(unpack_msrest_error(e))
 
 
+def iot_central_device_capability_model_get(
+    cmd, device_id, app_id, central_api_uri="api.azureiotcentral.com"
+):
+    provider = CentralDeviceProvider(cmd, app_id)
+    device_template = provider.get_device_template(device_id)
+    output = json.dumps(device_template, indent=4)
+    six.print_(output)
```

Comment:
It's for special cases where we print the result. You should return the Python object (which should be something that is compatible with json.dumps()). There are downstream ramifications if you don't. For example, being unable to use the CLI core --query capability.